### DISCUSSION TITLE
Ability to define user coordinate system for StructuralPointSupport - object extension

### DISCUSSION THREAD

**Discussion by sonkaste17** [StructuralAnalysisFormat/StructuralAnalysisFormat-Doc#271] [Ideas]
So far in SAF, Point support can act only in the global coordinate system even when it is applied "In node". We would like to extend the functionality with the possibility to define the user-defined coordinate system for a node and therefore the direction of constraints.

https://www.saf.guide/en/latest/supports-and-hinges/structuralpointsupport.html

**Comment by RickvanDijk1** [comment]
In addition to this; we would be very interested in also having (SCIA's) _Internal Nodes_ for Surfaces.

**Reply by sonkaste17** [reply]
Hello, thank you for your comment! 

Surfaces ([StructuralSurfaceMember](https://www.saf.guide/en/stable/structural-analysis-elements/structuralsurfacemember.html)) can have _Internal nodes_ listed in a property _Internal nodes_. Is that what you asked for or you are interested in the coordinate definition of _Internal nodes_ to be added?

**Reply by RickvanDijk1** [reply]
Brilliant! I expected it the other way around; as sometimes the local coordinates are relevant. However; this is exactly what I needed!
